mirascope/tools/system/_computer_use.py

```diff
@@ -0,0 +1,153 @@
+from __future__ import annotations
+
+import io
+import tarfile
+from abc import ABC
+from contextlib import suppress
+from typing import ClassVar
+
+import docker
+from docker.models.containers import Container
+from pydantic import Field
+
+from mirascope.core import toolkit_tool
+from mirascope.tools.base import (
+    ConfigurableTool,
+    ConfigurableToolKit,
+    _ToolConfig,
+    _ToolSchemaT,
+)
+
+
+class ComputerUseToolkitConfig(_ToolConfig):
```

Comment:
hmm, I think with Anthropic's new "Computer Use" release, this term is too overloaded, so we should be more specific. I think instead of "computer_use" and "ComputerUse" we should make it specific to Docket, so something like `DockerOperationToolKitConfig` and `BaseDockerOperationTool` and `DockerOperationToolKit`.

WDYT?

Reply:
I agree with you